code/game/machinery/computer/apc_control.dm

```diff
@@ -105,12 +105,14 @@
 				authenticated = TRUE
 				auth_id = "[ID.registered_name] ([ID.assignment])"
 				log_activity("logged in")
+				playsound(src, 'sound/machines/terminal_on.ogg', 50, 0)
```

Comment:
Should make these defines instead of copypasting the same lines 5,000,000 times, that way if we want to move the file or something we don't have to change tons of code around to do it.